internal/ethapi/api.go

```diff
@@ -680,6 +680,7 @@ func (s *BlockChainAPI) GetProof(ctx context.Context, address common.Address, st
 	// create the proof for the storageKeys
 	for i, hexKey := range storageKeys {
 		key, err := decodeHash(hexKey)
+		keyInt := (*hexutil.Big)(new(big.Int).SetBytes(key[:]))
```

Comment:
It's a bit unorthodox to do this operation, involving `key`, before checking/handling the `err`. It's safe, in this instance, but not "The Way" :)